nifi-registry/nifi-registry-core/nifi-registry-web-api/src/main/java/org/apache/nifi/registry/web/service/ServiceFacade.java

```diff
@@ -70,6 +70,8 @@ public interface ServiceFacade {
 
     Bucket deleteBucket(String bucketIdentifier, RevisionInfo revisionInfo);
 
+    Bucket migrateBucket(Bucket bucket);
```

Comment:
This could go away, or could become `createBucket(Bucket bucket, boolean preserveSourceProperties)`